dart/looker_sdk/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: looker_sdk
+version: 0.0.1
+environment:
+  sdk: '>=2.9.0 <3.0.0'
```

Comment:
Curious what this does. Does this mean use 2.9.x or greater but not 3.x?

Reply:
Again very old stuff. My recollection is that this was mainly driven by flutter but I believe your summation is correct.

Reply:
@bryans99  would we expect this SDK branch to be the best choice for using looker in flutter mobile apps, or platform calls to swift/kotlin SDKs, or rest api calls, or embedded urls, or just depending on things?